bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Transform.java

```diff
@@ -161,7 +169,7 @@ static float[] checkTransform(float[] elements) {
 
 @Override
 void destroy() {
-	Gdip.Matrix_delete(handle);
+	zoomLevelToHandle.values().forEach(handle  -> Gdip.Matrix_delete(handle));
```

Comment:
Nit: use method reference to improve readability (I personally prefer it specially when there's only 1 parameter)

```suggestion
	zoomLevelToHandle.values().forEach(Gdip::Matrix_delete);
```

Reply:
done